versions/3.0.md

```diff
@@ -1649,19 +1702,15 @@ definitions:
       example:
         $ref: http://foo.bar#/examples/name-example
 
-# in a request body, note the plural `examples` as the content-type is set by `consumes`:
+# in a request body, note the plural `examples` as the content-type is set to `*`:
```

Comment:
I'm unclear on "is set to `*`" as the example uses `application/json`. Also, this does not _set_ the Content-Type, it only _matches_ it.